### ISSUE TITLE
Print successful subtests

### ISSUE THREAD

**Issue by oliversheridanmethven** [pytest-dev/pytest-subtests#164]
Is there a flag or verbosity level I can set so I can see which tests were run, including the passing ones, broken down and enumerating all the subtests. Currently it only lists the top level test. So to double check all the subtests are running it would be great to list them out as running (to double check I have not accidentally forgot to iterate through my tests).

**Comment by CarlosRDomin** [comment]
+1, we use pytest with the `-rA` arg but this plugin only lists subtests with the "SUBSKIP" / "SUBFAIL" status, would be great to optionally also include "SUBPASS" outcomes

**Comment by CarlosRDomin** [comment]
Looked into it and pushed two commits that I believe fix the problem. The first one allows `unittest` subtests to print `,` upon success. The second one, includes successful subtests in the "short test summary info" section.

BEFORE:

![Image](https://github.com/user-attachments/assets/b9ab5019-9f0a-430b-b7ef-f56a1cd3ef1c)

AFTER: 

![Image](https://github.com/user-attachments/assets/a8c05f85-fce0-40aa-ab68-86143e769e0e)

Replicated with the following dummy test file:
```python
import unittest


class TestDummy(unittest.TestCase):
    def test_dummy(self):
        for i in range(8):
            with self.subTest(i=i):
                if i < 3:
                    self.skipTest("i needs to be at least 3")
                if i >= 6:
                    self.skipTest("i needs to be smaller than 7")
                self.assertGreaterEqual(i, 3)
```

CC @nicoddemus (assuming you're the maintainer)

**Comment by RonnyPfannschmidt** [comment]
How about opening a pull request

**Comment by CarlosRDomin** [comment]
Good point, done: https://github.com/pytest-dev/pytest-subtests/pull/190

**Comment by nicoddemus** [comment]
> (assuming you're the maintainer)

Yes, thanks!
